lib/logstash/inputs/syslog.rb

```diff
@@ -155,7 +154,7 @@ def tcp_listener(output_queue)
       socket = @tcp.accept
       @tcp_sockets << socket
 
-      break if @shutdown_requested.true?
+      break if stop?
```

Comment:
why not just having a ``` while !stop?```, I know you just changed one thing for the other, but looks more clean to use like a while, isn't?